src/components/ChangeLocation/index.js

```diff
@@ -13,6 +13,7 @@ const ChangeLocation = ({
   location,
   setLocation,
   setLocationPathDisplay,
+  modalHeaderTitle,
```

Comment:
you can have a default value here so that the existing pages using "Change Location" will not be affected